external-crates/move/crates/move-compiler/src/naming/translate.rs

```diff
@@ -1643,8 +1694,8 @@ fn module(
         functions: efunctions,
         constants: econstants,
     } = mdef;
-    context.current_package = package_name;
-    context.env.add_warning_filter_scope(warning_filter.clone());
+    let context = &mut Context::new(env, outer, package_name, ident);
+    context.add_warning_filter_scope(warning_filter.clone());
```

Comment:
Small nit: since the exit is called `pop`, should this be `push_warning_filter_scope`?